daemon/volumes.go

```diff
@@ -385,15 +385,14 @@ func copyExistingContents(source, destination string) error {
 // copyOwnership copies the permissions and uid:gid of the source file
 // into the destination file
 func copyOwnership(source, destination string) error {
-	var stat syscall.Stat_t
-
-	if err := syscall.Stat(source, &stat); err != nil {
+	stat, err := system.GetStat(source)
```

Comment:
The only place isFileOwner is used is here, btw:
https://github.com/docker/docker/blob/13bfbfc84bd63fd2014bc6590a8a568c0560d5f8/docker/daemon.go#L38